lib/node_modules/@stdlib/ndarray/README.md

```diff
@@ -77,6 +77,7 @@ In addition, the namespace contains the following multidimensional array utility
 -   <span class="signature">[`ndarrayDataTypes( [kind] )`][@stdlib/ndarray/dtypes]</span><span class="delimiter">: </span><span class="description">list of ndarray data types.</span>
 -   <span class="signature">[`ndemptyLike( x[, options] )`][@stdlib/ndarray/empty-like]</span><span class="delimiter">: </span><span class="description">create an uninitialized ndarray having the same shape and data type as a provided ndarray.</span>
 -   <span class="signature">[`ndempty( shape[, options] )`][@stdlib/ndarray/empty]</span><span class="delimiter">: </span><span class="description">create an uninitialized ndarray having a specified shape and data type.</span>
+-   <span class="signature">[`ndarray2array( dtype, buffer, shape, strides, offset, order[, options] )`][@stdlib/ndarray/to-array]</span><span class="delimiter">: </span><span class="description">fancy multidimensional array constructor.</span>
```

Comment:
@Planeshifter We're getting duplicates in the namespace ToCs. `@stdlib/ndarray/to-array` is also in L92. Similarly, L75 and L76 are the same.